packer/ansible/cyhy_dashboard.yml

```
@@ -0,0 +1,10 @@
+---
+- hosts: cyhy_dashboard
+  name: Install and configure cyhy-dashboard
```

I know you didn't write this, but might as well fix it since `cyhy-dashboard` isn't really a thing:
```suggestion
  name: Install and configure the CyHy dashboard
```